src/Builder/AbstractFormContractor.php

```diff
@@ -150,6 +157,18 @@ private function isAnyInstanceOf(?string $type, array $classes): bool
             }
         }
 
+        // handle form type inheritance and check all parent types
+        $resolvedType = $this->formRegistry->getType($type);
+        if (null !== $resolvedType->getParent()) {
```

Comment:
You have to create a variable 
$parent = $resolvedType->getParent();
if (null !== $parent)

Reply:
yeah will check it later